### DISCUSSION TITLE
WtFast,FlexQoS,and more...

### DISCUSSION THREAD

**Discussion by ZliTebra** [community/community#145841] [Other Feature Feedback, Questions, & Ideas]
The name of my Asus device says Gaming and there are not enough functions for that name.  The model is Tuf Gaming AX5400. I get happy with every new firmware and get disappointed even faster when I see that nothing is being done to improve gaming and have even more options for the game and settings.  Please can you concentrate a little on gaming and do something about it?  WtFast or FlexQos enabled for Tuf AX5400 would be a good start.The model I use is perhaps a little older, but it still costs 200 euros.Thank you very much.

**Comment by samus-aran** [comment]
Hi @ZliTebra ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this topic. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
